charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -1156,7 +1156,8 @@ def add_op_request_access_to_group(self, name, namespace=None,
 
     def add_op_create_pool(self, name, replica_count=3, pg_num=None,
```

Comment:
The param `replica_count` only makes sense for a 'replicated' pool type. So I think we should refactor a bit this part of code to avoid confusion. Basically have two methods `add_op_create_replicate_pool()` and `add_op_create_erasure_pool()`. Or perhaps we could add a param like `pool_type` ?

Reply:
Yes you are right.  I tend to overload when things are similar, but I really should not. Will push a second iteration.